src/width.rs

```diff
@@ -1139,6 +1145,7 @@ fn chunks(s: &str, width: usize) -> Vec<String> {
             continue;
         }
 
+        line.push_str(prefix);
```

Comment:
I assume you're implying that the string doesn't contains any other text?
But it may will.
And in such cases we would not like to make all lines being links.
Only the lines that are contains the actual link. (to be precise the content of the links)

```rust
    let data = [Distribution {
        name: format!(
            "here we get a debian link {}, right go and install it.",
            format_osc8_hyperlink("https://www.debian.org/", "Debian"),
        ),
        is_hyperlink: true,
    }];
```

```
".--------------------------------.\n| name            | is_hyperlink |\n| here we get a   | true         |\n| debian link  \u{1b}]8   |              |\n| ;;https://www.d |              |\n| ebian.org/      |              |\n| \u{1b}\\Debian \u{1b}]8;; \u{1b}\\         |              |\n| right go and    |              |\n| install it.     |              |\n'--------------------------------'"
```

Reply:
Addressed by 5950c102c2146c7b0d6f5b6ad78831c8ed5086d6